docker/docker-compose.yml

```diff
@@ -150,10 +150,10 @@ services:
       yarn:
         condition: service_started
 
-  # To transition from a previous major version to pg13, run ./docker/pg13/upgrade.db
+  # To transition from a previous major version to pg16, run ./docker/pg16/upgrade.sh
   db_previous:
     build:
-      context: docker/pg12
+      context: docker/pg13
```

Comment:
Maybe we don't want to change anything about the previous service (image or volume name) in this PR? I had to fiddle with volumes and the image to be able to boot this.

Reply:
Maybe defining an image on db_previous will fix this. Otherwise I guess we'd leave db_previous as pg12 and add a new db_13 service